guides/common/modules/proc_configuring-server-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,84 @@
+[id="Configuring-{project-context}-Server-to-Provision-{client-os-context}-on-Secure-Boot-Enabled-Hosts_{context}"]
+= Configuring {ProjectServerTitle} to Provision {client-os} on Secure Boot Enabled Hosts
+
+Secure Boot follows a chain of trust from the start of the host to the loading of Linux kernel modules.
+The first shim that is loaded determines which distribution is allowed to be booted or kexec'ed until next reboot.
+
+[IMPORTANT]
+====
+For each subnet you have to perform the following steps on the corresponding Smart Proxy that is configured as a TFTP proxy to be able to provision {client-os} on Secure Boot enabled hosts using the *Grub2 UEFI SecureBoot (target OS)* PXE loader.
```

Comment:
```suggestion
You have to perform the following steps on your {SmartProxy} that is configured as a TFTP proxy to provision {client-os} on Secure Boot enabled hosts.
```

alternative:

```suggestion
You have to perform the following configuration steps on each {SmartProxy} used as a TFTP proxy for the subnet of your Secure Boot enabled host.
```

Reply:
Thanks for your suggestions, @maximiliankolb. I kind of combined them both. What do you think:
```suggestion
You have to perform the following configuration steps on each {SmartProxy} used as a TFTP proxy for a subnet to provision Secure Boot enabled hosts on it.
```